src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -2457,6 +2642,22 @@ public void configure(Config config) {
       } else {
         this.iis = CwsServer.IIS.name().equalsIgnoreCase(server);
       }
+      try {
+        this.readTimeout = Integer.parseInt(
```

Comment:
The factory is used in init to authenticate and check the username/password, so it has to be configured before we do that. I'd hoped to avoid a long possibly-changing list of parameters and just let configure pull out the values it needed.